asset/dcr/wallet.go

```diff
@@ -79,8 +79,6 @@ func (w *Wallet) CloseWallet() error {
 	}
 
 	w.log.Info("Wallet closed")
-	w.mainWallet = nil
-	w.db = nil
```

Comment:
Not related to the problem but I don't understand why this needs to be done and I'm concerned it could cause panics.